router/src/components/router.rs

```diff
@@ -13,8 +13,6 @@ use leptos::{
     },
     *,
 };
-#[cfg(feature = "transition")]
```

Comment:
Feature removed in https://github.com/leptos-rs/leptos/commit/c0c42fdf597e52c24f5960a2b1b4d467849959da